### DISCUSSION TITLE
What can i do ?  Ask for help

### DISCUSSION THREAD

**Discussion by Poker-Pan** [lululxvi/deepxde#994] [Q&A] [answered]
I want to solve a Laplce equation with PINNs, but the right-hand source term is the case with singularities. How to solve such a problem is better, thank you！

**Comment by lululxvi** [comment, marked as answer]
Try to make singularity weaker. But how to do it? It is problem dependent.

**Reply by Poker-Pan** [reply]
OK, thank you very much for your help. I will try it
